### PR TITLE
Adding styling to the change password page.

### PR DESCRIPTION
**Pull request by SachiGoto**
# Description

- Added styling to the change password page.
- Added an eye icon to show and hide passwords - planning on incorporating it into the password fields on the sign-up and sign-in pages later.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## URL
http://localhost:3000/change-password
## Screenshot
| Screenshot 2024-02-02 at 23 29 03 | Screenshot 2024-02-02 at 23 28 52 |
| --- | --- |
| Screenshot 2024-02-02 at 23 28 43 | Screenshot 2024-02-02 at 23 28 32 |

### CODE REVIEW

**Review by YoshinariYamanaka — APPROVED** [review]

File: app/(auth)/change-password/page.tsx [hunk 1]
```diff
@@ -63,27 +65,53 @@ export default function ChangePassword() {
...
+      w="100vw"
+      minH={{ base: 'calc(100vh - 145px)', md: 'calc(100vh - 210px)' }}
+      color={color}
+    >
+      <VStack
+        mt={{ base: '20px', md: '40px' }}
+        mx="auto"
+        w={{ base: '380px', md: '480px' }}
+        px={{ base: '16px', md: '80px' }}
+      >
```
Comment (kanatagu) [inline app/(auth)/change-password/page.tsx]: Padding should be outside of the form? 
Also you need space 70px below the button like the Figma design
Screenshot 2024-02-04 at 11 33 49 AM

File: app/(auth)/change-password/page.tsx [hunk 2]
```diff
@@ -63,27 +65,53 @@ export default function ChangePassword() {
...
+          as="form"
+          onSubmit={changePasswordHandler}
+        >
           <FormControl isInvalid={!!errors.oldPassword}>
             <FormLabel>Old Password</FormLabel>
-            <Input type="password" {...register('oldPassword')} />
+            <InputForm
+              {...register('oldPassword')}
+              w={{ base: '358px', md: '480px' }}
+              showInput={true}
```
Comment (kanatagu) [inline app/(auth)/change-password/page.tsx]: Maybe it does make sense if you pass the props as "hasEyeIcon". `showInput={true}` sounds like you always show input, but you hide input after a user click the icon.
You can use the same password input in login form too!

Comment (SachiGoto) [reply app/(auth)/change-password/page.tsx]: That makes total sense! I changed the name!

File: app/(auth)/change-password/page.tsx (hunk 1 above)
Comment (SachiGoto) [reply app/(auth)/change-password/page.tsx]: Thank you for catching that! I added 70px paddings under the button and added left and right padding outside of the form!

Comment (SachiGoto) [reply app/(auth)/change-password/page.tsx]: Screenshot 1
Screenshot 2

File: app/components/input/input-form.tsx [hunk 3]
```diff
@@ -7,20 +8,24 @@ import {
...
     const borderColor = useColorModeValue('gray.300', 'gray.500')
     const placeholdercolor = useColorModeValue('gray.400', 'gray.600')
+    const [show, setShow] = useState(false)
+    const handleClick = () => setShow(!show)
 
     return (
       <InputGroup minW={'100%'}>
         <ChakraFormInput
           {...props}
+          type={show ? 'text' : props.type}
```
Comment (SachiGoto) [inline app/components/input/input-form.tsx:28]: There was a mistake in my code. It should pass a defined input type if 'show' is not true. Previously, it was only passing the type 'password,' causing all input fields to become of type password.

**Review by kanatagu — APPROVED** [review]

**Pull request merged**